Makes explicit that local executions are excluded from flex meters (#100)
hashicorp/terraform-docs-common: website/docs/cloud-docs/run/remote-operations.mdx

### Disabling Remote Operations

[execution_mode]: /cloud-docs/workspaces/settings#execution-mode

> [terraform_cloud_for_business]: https://cloud.hashicorp.com/docs/hcp/admin/billing/flex-billing#terraform-cloud-for-business

Reviewer: PMM is suggesting to rename header to Terraform Cloud Business Tier

Author: https://github.com/hashicorp/cloud.hashicorp.com/pull/458

Many of Terraform Cloud's features rely on remote execution and are not available when using local operations. This includes features like Sentinel policy enforcement, cost estimation, and notifications.

> You can disable remote operations for any workspace by changing its [Execution Mode][execution_mode] to **Local**. This causes the workspace to act only as a remote backend for Terraform state, with all execution occurring on your own workstations or continuous integration workers. Consequently, any [Apply][terraform_cloud_for_business] that is run in Local Execution mode will not count towards any Flex meters or Service Entitlements.

Reviewer 2: ```suggestion
You can disable remote operations for any workspace by changing its [Execution Mode][execution_mode] to **Local**. This change causes the workspace to act only as a remote backend for Terraform state, with all execution occurring on your own workstations or continuous integration workers. Consequently, any [Apply][terraform_cloud_for_business] that is run in Local Execution mode does not count towards any Flex meters or Service Entitlements.
```

Suggest adding "change" after "This" to avoid a "This" with no subject. Suggest changing "will not count" to "does not count" to avoid using future tense.

> You can disable remote operations for any workspace by changing its [Execution Mode][execution_mode] to **Local**. This change causes the workspace to act only as a remote backend for Terraform state, with all execution occurring on your own workstations or continuous integration workers. Consequently, any [Apply][terraform_cloud_for_business_tier] that is run in Local Execution mode does not count towards any Flex meters or Service Entitlements.

Reviewer 3: A couple of suggestions to make things more precise.  Also --> I noticed that "Flex" was capitalized, but meters was not. I fixed that, but pls let me know if that's wrong here :) 

```suggestion
You can disable remote operations for any workspace by changing its [Execution Mode][execution_mode] to **Local**. Local Execution causes the workspace to act only as a remote backend for Terraform state, with all Terraform runs occurring on your own workstations or continuous integration workers. Consequently, any [Apply][terraform_cloud_for_business_tier] operation that you run in Local Execution mode does not count towards any Flex Meters or Service Entitlements.
```
